DATA/UNIVERSE/SYSTEMS/LI03/li03.ini

```diff
@@ -3339,7 +3343,7 @@ ids_name = 459565
 pos = -65836, 0, -46697
 shape = SPHERE
 size = 10000,0,0
-property_flags = 129
+property_flags = 112
```

Comment:
Please revert this, seems accidental.

Reply:
Not accidental. I picked 112 thinking this and Copperton should be treated the same, but it shouldn't quite - since we're in a debris field here, the correct flag is 176, commit is coming.